package.json

```diff
@@ -12,6 +12,7 @@
     "fix": "yarn run eslint --fix \"src/**/*.ts\" \"test/**/*.ts\" && prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "build": "tsc -p tsconfig.json",
     "dev": "yarn run ts-node src/openapi-vuepress-markdown.ts",
+    "pretest": "tsc -p tsconfig.json",
```

Comment:
Out of scope of this PR I think?

Reply:
Rewire operates on the JavaScript files not typescript. Therefore tests depend on compilation.
In package.jsoj any preX script runs before X script.

Reply:
I see, makes sense